tsconfig.json

```diff
@@ -19,8 +19,5 @@
     "include": [
         "src/**/*.ts",
         "custom_typings/main.d.ts"
-    ],
-    "exclude": [
-        "node_modules"
     ]
```

Comment:
Reverted!

I suspect that atom-typescript will eventually fix this bug so we should recheck every once in a while. It's possible that they haven't added support for `include`, as it was added fairly recently in 2.0, and if it looks like you haven't told typescript what to include (via either `include` or `files` then it will just look for `**/*.[ts|js]`, which yeah, is super crazy slow)